core/const.py

```diff
@@ -649,14 +648,26 @@ class ITLRegion(StrEnum):
         "Project Details",
     ),
     "GIS Provided": ("Are you providing a GIS map (see guidance) with your return?", "Project Details"),
+    "Start Date": ("Start Date - mmm/yy (e.g. Dec-22)", "Projects Progress Summary"),
```

Comment:
Should the comment at the start of this section now say something like "Round 3 onwards" or "Round 3 and 4"?